.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+repos:
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: v1.4.4
+    hooks:
+    -   id: autopep8
```

Comment:
I've tested black for other projects, and find that it can format stylistically well-written pandas illegibly. I know black is increasingly popular in the community, and if I'm working on a project without extensive pandas and/or ORM usage, I think it's a nice tool, but in this case it can do some weird stuff.

Reply:
I considered it especially hard, because it's what pandas itself uses

Reply:
That's interesting. I haven't had any problems with pandas + black. Do you have some examples?

Reply:
[Example](https://black.now.sh/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AMMAUNdAD2IimZxl1N_Wg0-ASR35PBDdRWZoCW7oyk9wruYALBml7ddFGy0aIPjbTlPi-NJonr6W43iKWsLIb9fZt7_w9_9yPhohh-EzzmOGtmk4ufZV_eWu2TXZ1IBTTClWEsTzQNwQ1TLbMLettakyPdWzf9JquYIn5qZubISFOJbBtMgx4FI6MlMxBhwtp-SUB5OkpNW9ikl_Ig4lNlCxmK4Obw0E0ne9vekEpbfeiBMfEGI_tonAmAZfW_McngMY5kq9KF_WXJYnAcUC7mFAPMxgVe69KxDPeRb47QSavufwSis27bPd-4PKE7VZIx0hQwcosjCAOBx7WJaDWsU8g38wa-SPZoV8NxuS9MsRH4R9CQ69_L1UJNDBzUxvUZd37WoMBovb3sQl6QzAUHC8p2nkrZRUIUczdqb_CxG3TkQovKUYiqyAADTW7pmBaMRhwAB3wKNBgAAEztDQbHEZ_sCAAAAAARZWg==)